Makefile

```diff
@@ -246,8 +249,19 @@ yaml: manifests ensure-kustomize
 	$(KUSTOMIZE_BIN) build config/default > manifests/crd.yaml
 
 # Generate Go files from Chaos Mesh proto files.
+ifeq ($(IN_DOCKER),1)
```

Comment:
`ifneq`?

Reply:
No. `ifeq ($(IN_DOCKER),1)` means we need to run the following command in the container. And if we are on the host, we should use `docker run` to enter a container.